main.go

```diff
@@ -1,57 +1,43 @@
 package main
 
 import (
-	"flag"
+	"errors"
+	"fmt"
+	"html"
+	"io/fs"
 	"os"
+	"time"
 
-	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
+	"github.com/flashbots/go-template/config"
+	"github.com/flashbots/go-template/server"
 )
 
-var (
-	version = "dev" // is set during build process
-
-	// Default values
-	defaultDebug      = os.Getenv("DEBUG") == "1"
-	defaultLogProd    = os.Getenv("LOG_PROD") == "1"
-	defaultLogService = os.Getenv("LOG_SERVICE")
-
-	// Flags
-	debugPtr      = flag.Bool("debug", defaultDebug, "print debug output")
-	logProdPtr    = flag.Bool("log-prod", defaultLogProd, "log in production mode (json)")
-	logServicePtr = flag.String("log-service", defaultLogService, "'service' tag to logs")
-)
+var version = "dev" // is set during build process
 
 func main() {
-	flag.Parse()
-
-	logger, _ := zap.NewDevelopment()
-	if *logProdPtr {
-		atom := zap.NewAtomicLevel()
-		if *debugPtr {
-			atom.SetLevel(zap.DebugLevel)
+	cfg := config.NewServerConfig(version)
+
+	// Make sure to flush the logger before exiting the app
+	defer func() {
```

Comment:
we should create a `logging` or `logger` package in https://github.com/flashbots/go-utils

and this should be a function like

```go
logSync(log *zap.Logger)
```